### PR TITLE
Step 1: Add very basic CRUD API for Todos (in-memory approach)

### PR DESCRIPTION
**Pull request by vladislav-shkutovich**
Create: `/todos` **POST**
Read: `/todos` **GET**, `/todos/:id` **GET**
Update: `/todos/:id` **PATCH**
Delete: `/todos/:id` **DELETE**

Body example:
```
{
    "id": "new/updated id",
    "title": "new/updated todo title",
    "description": "new/updated todo description"
}
```

### CODE REVIEW

File: mocks/todos.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { TodoItem } from 'types/todos'
```
Comment (masontikhonov) [inline mocks/todos.ts]: Let's move this file to the corresponding module.

File: src/todos/todos.database.service.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { Injectable, NotFoundException } from '@nestjs/common'
+import { initialTodos } from 'mocks/todos'
+import { TodoItem } from 'types/todos'
+
+@Injectable()
+export class TodosDatabaseService {
+  static todos: TodoItem[] = initialTodos
```
Comment (masontikhonov) [inline src/todos/todos.database.service.ts]: Array is not the best choice of data structure when you need random access by unique ID and your data is unordered. Let's use `Map` instead

File: mocks/todos.ts (hunk 1 above)
Comment (vladislav-shkutovich) [reply mocks/todos.ts]: Fixed, thanks!

File: src/todos/todos.database.service.ts (hunk 2 above)
Comment (vladislav-shkutovich) [reply src/todos/todos.database.service.ts]: Fixed!

**Pull request merged**